assets/src/stories-editor/blocks/amp-story-text/edit.js

```diff
@@ -146,30 +211,41 @@ class TextBlockEdit extends Component {
 				</BlockControls>
 				<div className={ classnames( wrapperClass, {
 					'with-line-height': ampFitText,
+					'is-empty-draggable-text': ! isEditing && ! content.length,
 				} ) } style={ wrapperStyle } >
-					<RichText
-						wrapperClassName="wp-block-amp-story-text"
-						tagName="p"
-						// Ensure line breaks are normalised to HTML.
-						value={ content }
-						onChange={ ( nextContent ) => setAttributes( { content: nextContent } ) }
-						// The 2 following lines are necessary for pasting to work.
-						onReplace={ this.onReplace }
-						onSplit={ () => {} }
-						style={ {
-							color: textColor.color,
-							fontSize: ampFitText ? autoFontSize + 'px' : userFontSize,
-							textAlign: align,
-							position: ampFitText && content.length ? 'static' : undefined,
-						} }
-						className={ classnames( className, {
-							'has-text-color': textColor.color,
-							[ textColor.class ]: textColor.class,
-							[ fontSize.class ]: ampFitText ? undefined : fontSize.class,
-							'is-amp-fit-text': ampFitText,
-						} ) }
-						placeholder={ placeholder || __( 'Write text…', 'amp' ) }
-					/>
+					{ isEditing &&
+						<RichText
```

Comment:
Maybe it would make sense to move this to `DraggableText` as well actually.

Reply:
Could rename it to `DraggableRichText`. Would make a more complete component. Thoughts?

Reply:
Although maybe it would make the component confusing with that many attributes.

Reply:
I think for now we can keep it as-is.